packages/sirius-web/backend/sirius-web-application/src/main/resources/schema/duplicateobject.graphqls

```diff
@@ -0,0 +1,43 @@
+extend type Subscription {
+  duplicateDialogTreeEvent(input: DuplicateDialogTreeEventInput!): TreeEventPayload!
+}
+
+input DuplicateDialogTreeEventInput {
+  id: ID!
+  editingContextId: ID!
+  representationId: String!
+}
+
+extend type EditingContext {
+  containmentFeatureNames(containerId: ID!, containedObjectId: ID!): EditingContextContainmentFeatureNamesPayload!
```

Comment:
Inputs and payloads are to be used in the GraphQL schema only for mutations, just return the list of string directly:

```
extend type EditingContext {
  containmentFeatureNames(containerId: ID!, containedObjectId: ID!): [String!]!
}
```